src/settings/FinePolicy/components/EditSections/OverdueAboutSection/OverdueAboutSection.test.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+import '../../../../../../test/jest/__mock__';
+
+import OverdueAboutSection from './OverdueAboutSection';
+
+const mockedName = 'John Doe';
+const mockedDescription = 'My description!';
+
+jest.mock('react-final-form', () => ({
+  Field: jest.fn(({ component, label, name, id, ...rest }) => {
+    return (
+      <div>
+        <label htmlFor={id}>
+          {label}
+          {component({ name, id, ...rest })}
+        </label>
+      </div>
+    );
+  })
+}));
+
+describe('OverdueAboutSection', () => {
+  beforeEach(() => {
+    render(<OverdueAboutSection />);
+  });
+  const getItemByPlaceholder = (placeholder) => screen.getByPlaceholderText(placeholder);
+
+  it('should render label for input', () => {
+    expect(screen.getByLabelText('ui-circulation.settings.finePolicy.overdueFinePolicyName')).toBeTruthy();
+  });
+
+  it('should render input field', () => {
+    expect(getItemByPlaceholder('input placeholder')).toBeTruthy();
+  });
+
+  it('should render input element with "required" attribute', () => {
+    expect(getItemByPlaceholder('input placeholder')).toHaveAttribute('required');
+  });
+
+  it('should type text into input element correctly', () => {
+    userEvent.type(getItemByPlaceholder('input placeholder'), mockedName);
+    expect(getItemByPlaceholder('input placeholder')).toHaveValue(mockedName);
+  });
+
+  it('should render label for textarea', () => {
+    expect(screen.getByLabelText('ui-circulation.settings.finePolicy.description')).toBeTruthy();
+  });
+
+  it('should render textarea', () => {
+    expect(getItemByPlaceholder('textarea placeholder')).toBeTruthy();
+  });
+
+  it('should render textarea element without "required" attribute', () => {
```

Comment:
I guess this test is redundant, try to remove it and check if coverage was changed

Reply:
you are right. done